block-storage/src/main/scala/coop/rchain/blockstorage/finality/LastFinalizedStorage.scala

```diff
@@ -6,6 +6,10 @@ import coop.rchain.models.BlockHash.BlockHash
 trait LastFinalizedStorage[F[_]] {
   def put(blockHash: BlockHash): F[Unit]
   def get(): F[Option[BlockHash]]
+
+  /** Methods below are for migration of finalized block from LastFinalizedStorage to BlockDagStorage */
+  def requireMigration: F[Boolean]
+  def recordMigrationDone: F[Unit]
```

Comment:
This comment is more for general situations with migrations. Methods don't need to be defined on interface because usually migration will be done in some initialization stage where we have concrete instance anyway.